trunk/Admin/partials/loop-post-types.php

```diff
@@ -0,0 +1,12 @@
+<div class="admin-quickbar-postlist" data-post-type="<?php echo $postType->name; ?>">
+    <div class="admin-quickbar-post-type"><?php echo $postType->label; ?>
+        <a class="dashicons dashicons-plus add-new"
+                href="<?php echo admin_url( 'post-new.php' ) . '?post_type=' . $postType->name; ?>"></a>
+    </div>
+    <div class="admin-quickbar-postlist-inner">
+        <?php
+        // loop categories of current post-type
+        echo $categoryLoop;
```

Comment:
The actual loop should happen here. The required data should be passed fully prepared to the template.

Reply:
Think it is done